appverifier/src/main/java/com/android/mdl/appreader/VerifierApp.kt

```diff
@@ -26,14 +41,30 @@ class VerifierApp : Application() {
         DynamicColors.applyToActivitiesIfAvailable(this)
         userPreferencesInstance = userPreferences
         Logger.setDebugEnabled(userPreferences.isDebugLoggingEnabled())
+        trustManagerInstance = trustManager
+        certificateStorageEngineInstance = certificateStorageEngine
+        certificateStorageEngineInstance.enumerate().forEach{
+            val certificate = parseCertificate(certificateStorageEngineInstance.get(it)!!)
+            trustManagerInstance.addCertificate(certificate)
+        }
```

Comment:
Where do we add all the built-in certificates?

Reply:
See a reply on a previous comment:

> I now separated the storage and the TrustManager. However, I do not add the certificates from the resources directly to the TrustManager. In the settings there is an option to copy all certificates from resources to the trustmanager (and storage). This makes it possible to test the application also without any root certificate (something we discussed before)

This is done in com.android.mdl.appreader.settings.CaCertificatesFragment.copyCertificatesFromResources() (line 115) and can be undone with deleteAllCertificates() (line 135)

Reply:
Why aren't we just adding them at application startup? That way it's much easier to figure out which ones we start with and we also get to fail early if one of the certificates are bad....

Reply:
Ok, and should we remove the function to add them to the storage?

Reply:
Correct, we should not import them and add them to storage. Ideally they would show up with some metadata in the UI so they can't be deleted either... let me take a look at the branch to see how it works.